### ISSUE TITLE
Golem doesnt seem to have installed properly- an odd bug

### ISSUE THREAD

**Issue by stonercough** [golemfactory/clay#1139]
tl;dr my account has 1000 gnt after about 2 hours of mining. i first installed right before. However, if i ever checked the network status through the golemcli, it says 'Not connected to Golem Network, check seed parameters'. (and said that when i first discovered this) 

'network show' gives an empty chart, as well as 

I installed golem this morning, and only after futzing with the dependencies for an hour or two did it actually work. I got the archive file to work, and i ran the program as the instructions said, with './golemapp --no-gui' 
and it gives me an error: option not found. However, running './golemapp' works. 'sudo ./golemcli' also works, and at this point i had an errand to run, so i left my pc as it was and left the program running for a few hours. I come back i have 1k coin! I immediately knew it had to be a bug. But i have absolutely no way to check if the coins are a bug or not; i dont know how to transfer them from the wallet (presumably the long key that pops up when i enter 'account')

I haven't done much to it other than leave the first terminal window open and explore the command line options. I haven't changed any settings from default. 

(in case times seemed fucked, i didn't hit 'submit' last night.

**Comment by stonercough** [comment]
Another thing: I left the app running all night, no additional GNT. I'm completely sure its an account bug now, no doubts. 

How do i fix my account? why can't i change my node_name? (can i keep this gnt pls)

**Comment by badb** [comment]
When you connect to our test network the first time you receive tGNT from our faucet. Keep in mind that they are not “real” GNT from main chain and can be only used in test-network.
 
You can’t earn any additional tGNT because you are not connected to Golem network. There most probably l reasons for that:You don’t have public IP and you didn’t forwarded proper ports on your router. Check this instruction if that’s the issue: https://github.com/golemfactory/golem/wiki/Testing#how-to-start-testing
 
Also, the option is called `--nogui` that's why `--no-gui` was not found.
